app/test_split_integer.py

```diff
@@ -2,20 +2,46 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 20
+    number_of_parts = 3
+
+    assert sum(split_integer(value, number_of_parts)) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 21
+    number_of_parts = 3
+
+    result = split_integer(value, number_of_parts)
+
+    if value % number_of_parts:
```

Comment:
based on the test's name, you're only testing the scenario where value is divisible by number_of_parts, so the if condition is not needed.